invokeai/frontend/web/src/services/api/types.ts

```diff
@@ -132,6 +132,7 @@ export type CompelInvocation = s['CompelInvocation'];
 export type DynamicPromptInvocation = s['DynamicPromptInvocation'];
 export type NoiseInvocation = s['NoiseInvocation'];
 export type DenoiseLatentsInvocation = s['DenoiseLatentsInvocation'];
+export type RescaleLatentsInvocation = s['ResizeLatentsInvocation'];
```

Comment:
this seemed to be missing despite being in the schema so I added it

Reply:
I think you are after either `ScaleLatentsInvocation` or `ResizeLatentsInvocation` (not `RescaleLatentsInvocation`)?

Reply:
renamed